### ISSUE TITLE
Add CONFIG_DEBUG_INFO_BTF=y in edge

### ISSUE THREAD

**Issue by alban** [flatcar/Flatcar#13]
This option is available from Linux 5.2. It would allow us to try BTF and CO-RE (BPF compile once, run everywhere).

**Comment by dongsupark** [comment]
Filed a PR https://github.com/flatcar-linux/coreos-overlay/pull/161

**Comment by dongsupark** [comment]
The BTF kernel config has been included in Flatcar Edge, since 2411.99.0.
Though I would like to recommend 2430.99.0 or newer, due to other bug fixes.

```
Flatcar Container Linux by Kinvolk edge (2430.99.0)
core@localhost ~ $ zgrep BTF /proc/config.gz
CONFIG_DEBUG_INFO_BTF=y
```
